client/mass/test/matrix.integration.spec.js

```diff
@@ -370,9 +370,9 @@ tape('geneVariant terms and dictionary terms', function (test) {
 						{
 							name: '',
 							lst: [
-								{ term: { name: 'TP53', type: 'geneVariant', isleaf: true } },
-								{ term: { name: 'KRAS', type: 'geneVariant', isleaf: true } },
-								{ term: { name: 'AKT1', type: 'geneVariant', isleaf: true } },
+								{ term: { gene: 'TP53', name: 'TP53', type: 'geneVariant', isleaf: true } },
```

Comment:
can isolate these 3 genes into a variable and do `lst:[ ...structuredClone(genes), ...]`

Reply:
Ok will do